src/constant/index.js

```diff
@@ -150,10 +150,10 @@ export const LIKECOIN_NFT_ABOUT_ITEMS = IS_TESTNET
       'likenft1yhsps5l8tmeuy9y7k0rjpx97cl67cjkjnzkycecw5xrvjjp6c5yqz0ttmc',
     ];
 
-// 1. Set undefined to disable view content directly for all NFT classes
-// 2. Set empty array to enable view content directly for all NFT classes
+// 1. Set null to enable view content directly for all NFT classes
+// 2. Set empty array to disable view content directly for all NFT classes
 // 3. Set array of NFT class Id to disable view content directly for specific NFT classes
-export const LIKECOIN_NFT_DIRECT_VIEW_CONTENT = IS_TESTNET ? [] : undefined;
+export const LIKECOIN_NFT_DIRECT_ACCESS_ITEMS = IS_TESTNET ? [] : null;
```

Comment:
do we want to flip this? now it is enable nothing for testnet and all for production

Reply:
My mistake... updated